src/test/java/picard/sam/SplitSamByNumberOfReadsTest.java

```diff
@@ -251,4 +252,21 @@ private void compareInputWithOutputs(final SamReader reader1, final SamReader re
             Assert.assertEquals(rec.getReadName(), inputRec.getReadName());
         }
     }
+
+    @Test
+    public void testOutPrefixWithZeros() throws IOException {
```

Comment:
this test will generate two outputs, right? Please delete the second one as well.

Reply:
(and while you're at it, make sure it's valid...)